action.yml

```diff
@@ -16,10 +16,6 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - run: |
-        python3 -m pip install --user pipx
-        python3 -m pipx ensurepath
-      shell: bash
```

Comment:
I suppose this technically counts as a breaking change, but that should be alright, I think. Thank you for the contribution.